HISTORY.md

```diff
@@ -1,5 +1,8 @@
 ### ensmallen ?.??.?: "???"
 ###### ????-??-??
+ * Fix the current CMAES inconsistencies
+   ([#351](https://github.com/mlpack/ensmallen/pull/351)).
```

Comment:
this doesn't look right.  the present PR is 367, and has the description of "Implementation of Active CMAES".  why mention 351 here?

Reply:
Sorry about that! The branch of this PR (367) is actually a branch of PR 351's branch 
😅. I was trying to find a cleaner way to open this PR, but couldn't really find one (that being said, is there a way?). 
I know I'll have to deal with the merge conflicts if PR 351 gets merged.

Reply:
I'll update this soon too.

Reply:
suggest to keep PR 351 and this PR as separate (ie. not inter-dependent).  if this PR is merged first, PR 351 may then need to be updated (and vice-versa).

Reply:
I understand, I believe (between the two if any) PR 351 will get merged first. So I would have to update this PR soon (resolving merge and conflicts and such).